### PR TITLE
Fixed the bug on filer

### PR DESCRIPTION
**Pull request by Dilandds**
---
category: Bug fixes
authors: Dilandds
---
Issue number #1594 
Fixed the Minor visual regression in filters menu: "the amount select box has 2x borders around it"
By overwriting the css on the Select component

### CODE REVIEW

**Comment by MatissJanis** [comment]
👋 Feel free to ignore the failing VRT CI job (for now). I'm working on a fix for it here: https://github.com/actualbudget/actual/pull/1590

File: packages/desktop-client/src/components/filters/FiltersMenu.js [hunk 1]
```diff
@@ -181,7 +181,8 @@ function ConfigureField({
         <View style={{ marginBottom: 10 }}>
           <Stack direction="row" align="flex-start">
             {field === 'amount' || field === 'date' ? (
-              <Select
+              <Select style={{ borderWidth: 1, borderColor: 'transparent' }}
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/filters/FiltersMenu.js:184]: **💬 suggestion:** ‏I think you can just remove the `style` proper entirely. That should solve the issue.

File: packages/desktop-client/src/components/filters/FiltersMenu.js [hunk 2]
```diff
@@ -181,7 +181,8 @@ function ConfigureField({
         <View style={{ marginBottom: 10 }}>
           <Stack direction="row" align="flex-start">
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/filters/FiltersMenu.js:182]: And also I think we can remove the `Stack` wrapping component and the `<View style={{ flex: 1 }} />`.

File: packages/desktop-client/src/components/filters/FiltersMenu.js (hunk 1 above)
Comment (Dilandds) [reply packages/desktop-client/src/components/filters/FiltersMenu.js:184]: Thank you for your feedback, you mean the style property of the View component?

File: packages/desktop-client/src/components/filters/FiltersMenu.js (hunk 2 above)
Comment (Dilandds) [reply packages/desktop-client/src/components/filters/FiltersMenu.js:182]: Thank you for your feedback, are you suggesting to remove both <View> and <Stack> components?

Comment (MatissJanis) [reply packages/desktop-client/src/components/filters/FiltersMenu.js:182]: Just the `View` component can be removed and the `style` prop on `Select`. I think that would solve the problem (but please double check).

Thanks again!

**Pull request closed without merging**